cmd/podman/containers/ps.go

```diff
@@ -78,7 +78,7 @@ func listFlagSet(cmd *cobra.Command) {
 	flags := cmd.Flags()
 
 	flags.BoolVarP(&listOpts.All, "all", "a", false, "Show all the containers, default is only running containers")
-	flags.BoolVar(&listOpts.Storage, "external", false, "Show containers in storage not controlled by Podman")
+	flags.BoolVar(&listOpts.External, "external", false, "Show containers in storage not controlled by Podman")
```

Comment:
This is being normalized so Storage is also accepted, right?

Reply:
Yes that happened in an earlier PR.
https://github.com/containers/podman/blob/master/cmd/podman/utils/alias.go#L24-L25